### PR TITLE
Better exporting of Flight Flagger components

### PR DESCRIPTION
**Pull request by jacksonhyde**
- Adds a callback to Flight Flagger filters tied to the show/hide toggle
- Improves export heirarchy

### CODE REVIEW

File: src/components/FlightFlagger/FlightFlaggerTable.tsx [hunk 1]
```diff
@@ -36,16 +46,51 @@ export const FlightFlaggerTable = ({flights, ...others}: IFlightFlaggerTable) =>
...
+          </StyledTooltip>
+        </StyledTableHeaderCell>
         <StyledTableHeaderCell align="center">Exp PCP</StyledTableHeaderCell>
         <StyledTableHeaderCell align="center">Exp PCP Pax</StyledTableHeaderCell>
-        <StyledTableHeaderCell align="center">e-Gate</StyledTableHeaderCell>
-        <StyledTableHeaderCell align="center">EEA</StyledTableHeaderCell>
-        <StyledTableHeaderCell align="center">Non-EEA</StyledTableHeaderCell>
+        <StyledTableHeaderCell align="center">
+          e-Gate
+          <StyledTooltip title="Expeced total pax">
```
Comment (richbirch) [inline src/components/FlightFlagger/FlightFlaggerTable.tsx:72]: Minor typo here and below 'Expeced'

Comment (richbirch) [reply src/components/FlightFlagger/FlightFlaggerTable.tsx:72]: Oh actually I guess this should say 'Expected e-Gate pax', and below EEA & Non-EEA - though I also just realised this component isn't going to be used yet and we'll need to pass the splits dynamically so I guess it's fine to leave for now

**Review by richbirch — APPROVED** [review]
Looks good to me!

File: src/components/FlightFlagger/FlightFlaggerTable.tsx (hunk 1 above)
Comment (jacksonhyde) [reply src/components/FlightFlagger/FlightFlaggerTable.tsx:72]: I'll fix the typo! But yeah this component will need some work before we can use it.

**Pull request merged**